src/config/index.ts

```diff
@@ -0,0 +1,32 @@
+
+
+export enum GameIds {
```

Comment:
what if intead of having this list here, that would require that every time we have a new game we need to make an update from the lib, we pass the game-id string to the library init ?

I'm already doing that and storing it in the Cache.config inside the sdk file.
And keep this config logic with the environments and server configs agnostic to the game ids
https://github.com/decentraland/sdk-mini-games/blob/main/src/index.ts#L9-L11

Reply:
```ts
// inside the library

const { config } = getSDK()
const GAME_ID = config.gameId
fetch(someEndpoint(GAME_ID))
```

Reply:
thanks gon! i close this PR and create a new one